flightcheck/appHook.js

```diff
@@ -68,27 +72,19 @@ class AppHook {
     }
   }
 
-  // TODO: clean up whitespace removal code
   issue () {
-    let template = issue.render(`${this.path}/${this.mark}`, this)
+    const template = render(path.join(this.path, this.mark), this)
 
-    template = template.split('\n')
-    let title = template.find(string => string !== '')
-
-    template.splice(0, template.indexOf(title) + 1)
-
-    template = template.join('\n')
+    template.body += `\n<!-- Houston version ${config.houston.version} in ${config.env} environment -->`
```

Comment:
Yeah I think we should note that down as a additional feature for logging, I'm filing an issue for that